src/event.rs

```diff
@@ -317,6 +317,30 @@ pub enum WindowEvent<'a> {
         modifiers: ModifiersState,
     },
 
+    /// Touchpad magnification event with two-finger pinch gesture.
+    ///
+    /// Positive delta values indicate magnification (zooming in) and
+    /// negative delta values indicate shrinking (zooming out).
+    ///
+    /// At the moment, only supported on macOS.
+    TouchpadMagnify {
+        device_id: DeviceId,
+        delta: f64,
+        phase: TouchPhase,
+    },
+
+    /// Touchpad rotation event with two-finger rotation gesture.
+    ///
+    /// Positive delta values indicate rotation to left (counterclockwise) and
+    /// negative delta values indicate rotation to right (clockwise).
```

Comment:
Nit: "counterclockwise" and "clockwise" are the correct and precise terms for this, just always use those.